reposilite-site/data/guides/migration/artifactory.md

```diff
@@ -0,0 +1,38 @@
+---
+id: artifactory
+title: Artifactory
+---
+
+### Instructions
+
+1. Export Each Repository from Artifactory.
+    Be sure to check the `.m2 compatible export` box to get an export that reposilite can use. Also, we recommend excluding artifactory build metadata if your version of Artifactory has this feature.
```

Comment:
Below the 1st paragraph you could attach an image how this menu looks like in Artifactory with selected `.m2` compatibility :)